shared/js/l10n.js

```diff
@@ -1492,6 +1511,7 @@
         return getDirection(navigator.mozL10n.ctx.supportedLocales[0]);
       }
     },
+    gaiaVersion: '2.2',
```

Comment:
One more thought about the second point above:  I think it gives us time to work out a good solution for versioning in 2.3.  Worst case scenario, we'll put `deviceinfo.os` in a `meta` in 2.3 or we'll hardcode it in l10n.js and we will disable 2.2 langpacks in 2.3 apps this way, if we decide that's what we want to do.